plugins/Denizen/scripts/baguette.dsc

```diff
@@ -0,0 +1,37 @@
+baguette:
+    type: item
+    display name: <&f>Baguette
+    material: bread
+    flags:
```

Comment:
I don't think this type of item should be stackable due to it's 'reusable' nature, if you had a stack of these and used one then the following script would appear to break. Consider adding a uniqueifier to the flags to keep these from stacking.